.github/workflows/shortint_benchmark.yml

```diff
@@ -77,6 +77,20 @@ jobs:
           --name-suffix avx512 \
           --throughput
 
+      - name: Run multi-bit benchmarks with AVX512
+        run: |
+          rm -rf target/criterion
+          make AVX512_SUPPORT=ON bench_shortint_multi_bit
```

Comment:
I don't know if this is the right PR for this, as there were some slow downs/some runs took longer as we did not have enough cores

Reply:
What do you mean? I've made several clear commits in this PR , you want me to move the last commit to a new PR ?

Reply:
I think so, adding the decompression is easy, while there may be issues with the multi bit benchmarks ? Ah but you added them only to shortint, well if the timings are not crazy on shortint then should be fine !

Reply:
I remove the multi-bit shortint in the benchmark for now.